adwords_client/adwords_api/managed_customer_service.py

```diff
@@ -5,9 +5,12 @@ class ManagedCustomerService(cm.BaseService):
     def __init__(self, client):
         super().__init__(client, 'ManagedCustomerService')
 
-    def get_customers(self, client_customer_id=None):
+    def get_customers(self, client_customer_id=None, predicate=None):
         self.prepare_get()
         self.helper.add_fields('CustomerId', 'Name')
+        if predicate:
+            for predicate_item in predicate:
+                self.helper.add_predicate(predicate_item['field'], predicate_item['operator'], predicate_item['values'])
```

Comment:
No need to check for `if predicate`:

```python
for predicate_item in predicate or []:
    ...
```
It will iterate 0 times as the `[]` is empty.